scrape.py

```diff
@@ -514,6 +516,22 @@ def add_ebs_info(instances):
         by_type[instance_type].ebs_max_bandwidth = ebs_max_bandwidth
 
 
+def check_ebs_as_nvme(instances):
+    """Note which instances expose EBS as NVMe devices
+
+    Some of the new instances (like i3.metal and c5d family) will expose EBS
+    volume at /dev/nvmeXn1
+    https://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/nvme-ebs-volumes.html
+    """
+    instance_type = 'i3.metal'
+    families = ['c5', 'c5d', 'm5', 'm5d', 'r5', 'r5d', 'z1d']
```

Comment:
I think this is going to be the norm for any instance types launched going forward.

I would create a list of the legacy instance types and set it true for anything else.

Reply:
Are you sure? I think this guessing should be a little more cautious. Not all instances support NVMes or even SSDs.

Reply:
How about parsing it off the page there? The markup looks a little odd but maybe they'll keep the data updated.